### ISSUE TITLE
CocoaPods integration

### ISSUE THREAD

**Issue by skywinder** [Mozharovsky/CVCalendar#25]
Please, add your project to CocoaPods repo for easier integration

**Comment by Mozharovsky** [comment]
Hello @skywinder, 

CVCalendar is actually committed to CocoaPods trunk. You can install it either through adding files directly to your target or via CocoaPods using the following script. 

```Ruby
target 'YourProjectHere' do
     pod 'CVCalendar', '~> 1.0'
end
```

Once the documentation on CVCalendar is updated this method will be described. 

Hope it helps. 

Regards, 
Eugene

**Comment by skywinder** [comment]
Great, thanks! :+1:

**Comment by Mozharovsky** [comment]
Resolved #111
